### PR TITLE
[MIG] [11.0] Migrate hr_holidays_compute_days to 11.0

### PR DESCRIPTION
**Pull request by feketemihai**
Depends on:
- [ ] hr_public_holidays - #401

### CODE REVIEW

**Comment by feketemihai** [comment]
Depends on #401

File: hr_holidays_compute_days/models/hr_employee.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
-# -*- coding: utf-8 -*-
 # ©  2015 iDT LABS (http://www.@idtlabs.sl)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from datetime import datetime, time
-from openerp import models, api
+from odoo import models, api
```
Comment (novawish) [inline hr_holidays_compute_days/models/hr_employee.py]: import must follows alphabetical order

File: hr_holidays_compute_days/models/hr_holidays.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
 # ©  2015 iDT LABS (http://www.@idtlabs.sl)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields, api, _
-from openerp.exceptions import ValidationError
+from odoo import models, fields, api, _
```
Comment (novawish) [inline hr_holidays_compute_days/models/hr_holidays.py]: import must follows alphabetical order

File: hr_holidays_compute_days/models/hr_holidays_status.py [hunk 3]
```diff
@@ -1,8 +1,7 @@
-# -*- coding: utf-8 -*-
 # ©  2015 iDT LABS (http://www.@idtlabs.sl)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from odoo import models, fields
```
Comment (novawish) [inline hr_holidays_compute_days/models/hr_holidays_status.py]: import must follows alphabetical order

**Review by novawish — CHANGES_REQUESTED** [review]

**Comment by feketemihai** [comment]
The module has been incorporated in hr_public_holidays.

**Pull request closed without merging**